### PR TITLE
Update apache_camel.json

### PR DESCRIPTION
**Pull request by aashnajena**
@zregvart is a maintainer of the [Camel Website repository](https://github.com/apache/camel-website) and can approve these changes.

These changes are made for the following reasons :
- To include custom selectors for different start_urls
- To avoid duplicate records
- To index only content-level records

### CODE REVIEW

File: configs/apache_camel.json [hunk 1]
```diff
@@ -1,20 +1,58 @@
...
   "selectors": {
-    "lvl0": "article h1",
-    "lvl1": "article h2",
-    "lvl2": "article h3",
-    "lvl3": "article h4",
-    "lvl4": "article h5",
-    "lvl5": "article h6",
-    "text": "article p, article li"
+    "default": {
+      "lvl0": { "selector": "", "default_value": null },
```
Comment (s-pace) [inline configs/apache_camel.json]:
```suggestion
      "lvl0": { "selector": "", "default_value": "" },
```
default_value can not be `null`

File: configs/apache_camel.json [hunk 2]
```diff
@@ -1,20 +1,58 @@
...
+      "lvl1": "div.toolbar ul li:nth-child(1) a",
+      "lvl2": "div.toolbar ul li:nth-child(2) a",
+      "lvl3": "div.toolbar ul li:nth-child(3) a",
+      "lvl4": "div.toolbar ul li:nth-child(4) a",
+      "lvl5": "article h2",
+      "lvl6": "article h3",
+      "text": "article p, article li"
+    },
+    "blog": {
+      "lvl0": { "selector": "", "default_value": null },
```
Comment (s-pace) [inline configs/apache_camel.json]: Same feedback here. Why don't you want to define the `lvl0`? It helps to provide more context to your end users

Comment (aashnajena) [reply configs/apache_camel.json]: I have changed this to include a selector here.

File: configs/apache_camel.json [hunk 3]
```diff
@@ -1,20 +1,85 @@
...
+      "lvl2": "div.toolbar ul li:nth-child(2) a",
+      "lvl3": "div.toolbar ul li:nth-child(3) a",
+      "lvl4": "div.toolbar ul li:nth-child(4) a",
+      "lvl5": "article h2",
+      "lvl6": "article h3",
+      "text": "article p, article li"
+    },
+    "blog": {
+      "lvl0": "div.nav-panel-explore .version.is-current a",
+      "lvl1": {
```
Comment (s-pace) [inline configs/apache_camel.json:70]: It is not recommended to define a lvl0 & lvl2 while not defining a lvl1.

Comment (aashnajena) [reply configs/apache_camel.json:70]: We want the lvl1 entry to be "Blog". That's why we have specified an empty selector and a default value. Is there another way to do this?

Comment (s-pace) [reply configs/apache_camel.json:70]: I would recommend you[ to set both lvl0 and lvl1 global then](https://docsearch.algolia.com/docs/config-file/#using-global-selectors).

Comment (aashnajena) [reply configs/apache_camel.json:70]: We don't have the title "Blog" mentioned in any of our blog posts, so we cannot retrieve it from HTML markup.

Comment (s-pace) [reply configs/apache_camel.json:70]: Yes understood, this is why we recommend to use global selectors so that all hits contain this level

Comment (aashnajena) [reply configs/apache_camel.json:73]: Is this better now?

File: configs/apache_camel.json [hunk 4]
```diff
@@ -1,20 +1,89 @@
...
+      "lvl4": "div.toolbar ul li:nth-child(4) a",
+      "lvl5": "article h2",
+      "lvl6": "article h3",
+      "text": "article p, article li"
+    },
+    "blog": {
+      "lvl0": {
+        "selector": "div.nav-panel-explore .version.is-current a",
+        "global": true
+      }
```
Comment (s-pace) [inline configs/apache_camel.json:72]:
```suggestion
      },
```

**Comment by s-pace** [comment]
@aashnajena & @zregvart I have updated the configuration to enhance the search experience.

We're happy to provide DocSearch free of charges for any documentation website. This project is built to help the community. [As part of our attribution policy](https://docsearch.algolia.com/docs/styling/#attribution), we do ask our user to display our logo from the dropdown. Can you make sure to display the logo from the dropdown please? You [can find the assets here](https://www.algolia.com/press/?section=brand-guidelines). If you are using [InstantSearch under the hood, you can leverage its API quickly](https://www.algolia.com/doc/api-reference/widgets/powered-by/js/)

**Comment by zregvart** [comment]
@s-pace thank you, we indeed use our own presentation for the search results, and the reason for this is that when we initially implemented the search we did not use the Algolia's JavaScript package as it nearly tripled the page size. So we implemented our own AJAX and UI on top of the Algolia API, we did this in quite some haste omitting the attribution. We're now working with two interns from Outreachy (@aashnajena being one of them) to reimplement the search and one of tasks is to add the Algolia attribution.

**Pull request merged**